packages/sakuli-cli/src/commands/enable-enterprise/get-bootstrap-tasks.function.ts

```diff
@@ -68,5 +68,15 @@ export const getBootstrapTasks = (answers: EnterpriseAnswers): Task[] => {
     );
   }
 
+  if (answers.features.includes(FeatureChoices.Prometheus)) {
+    tasks.push(
+      ...getPackageBootstrapTasks("@sakuli/forwarder-prometheus", {
+        "sakuli.forwarder.prometheus.enabled": configValue("true"),
+        "sakuli.forwarder.prometheus.api.host": configValue(),
+        "sakuli.forwarder.prometheus.api.port": configValue("9091"),
```

Comment:
It looks like you did not implement a test for setting the port value but I suggest removing this default value to remain consistent with the other forwarder configuration, which only sets the `enabled` property to `true`.